src/vsl/parser/nodes/typeAlias.js

```diff
@@ -1,27 +0,0 @@
-import Node from './node';
```

Comment:
hm, we were going to use them as a variation of variable declarations:

```swift
public typealias GoatBarn = Barn<Goat>
```

maybe leave it and we can finish up in future PR? We could also just re-add it in the future too